openy.packages.yml

```diff
@@ -71,6 +71,14 @@ y_layout_builder:
   - ws_lb_tabs
   - lb_testimonials
   - lb_webform
+  - y_camp
+  - y_facility
+  - y_lb_main_menu_cta_block
+  - ws_home_branch
```

Comment:
This enables home branch for Standard, while it was never enabled there, because Small Ys are not using multiple branches.
So your suggestion here is appreciated

Reply:
@podarok I think Home Branch would make more sense for the advanced install and not the custom
```suggestion
```